vike/node/plugin/plugins/importUserCode/v1-design/getVikeConfig.ts

```diff
@@ -1111,20 +1116,48 @@ async function findPlusFiles(
     assertPosixPath(dir)
     ignorePatterns.push(`${path.posix.relative(userRootDir, dir)}/**`)
   }
-  const result = await glob(`**/+*.${scriptFileExtensions}`, {
-    ignore: [
-      '**/node_modules/**',
-      // Allow:
-      // ```
-      // +Page.js
-      // +Page.telefunc.js
-      // ```
-      '**/*.telefunc.*',
-      ...ignorePatterns
-    ],
-    cwd: userRootDir,
-    dot: false
-  })
+
+  let result: string[] = []
+
+  // If there is no .git folder, this will error
+  try {
+    const gitIncludePatterns = scriptFileExtensionList.map((ext) => `"**/+*.${ext}"`)
+    // -o lists untracked files only(but using .gitignore because --exclude-standard)
+    // -s adds the tracked files to the output
+    const { stdout } = await execA(
+      `git ls-files ${gitIncludePatterns.join(' ')} -os --exclude-standard --exclude="**/node_modules/**"`,
```

Comment:
Why do we need to exclude `node_modules`? Isn't it in `.gitignore` anyway?

Reply:
My mistake, I understand, it's for the case in which there is no `.gitignore` in the repo :+1: